show/main.py

```diff
@@ -1383,8 +1391,20 @@ def runningconfiguration():
 @click.option('--verbose', is_flag=True, help="Enable verbose output")
 def all(verbose):
     """Show full running configuration"""
-    cmd = "sonic-cfggen -d --print-data"
-    run_command(cmd, display_cmd=verbose)
+    cmd = ['sonic-cfggen', '-d', '--print-data']
+    stdout = get_cmd_output(cmd)
+
+    try:
+        output = json.loads(stdout)
+    except Exception as e:
```

Comment:
>Exception [](http://example.com/codeflow?start=11&length=9)

Do not catch general exception type in normal practice.. Could you use a more specific type?